site/app/models/notebook/Notebook.php

```diff
@@ -171,8 +173,10 @@ public function getMostRecentNotebookSubmissions(int $version, array $new_notebo
                             $recentSubmission = $this->getRecentSubmissionContents($notebookVal['filename'], $version, $student_id);
                             //get answer for the selected display version
                             $question_name = $notebookVal['filename'];
-                            $file = fopen("/var/local/submitty/courses/" . $semester . "/" . $course . "/submissions/" . $gradeable_id . "/" . $student_id . "/" . $display_version . "/" . $question_name, "r");
-                            $version_answer = fread($file, filesize("/var/local/submitty/courses/" . $semester . "/" . $course . "/submissions/" . $gradeable_id . "/" . $student_id . "/" . $display_version . "/" . $question_name));
+                            $file_path = $this->core->getConfig()->getCoursePath();
+                            $file_path = $file_path . "/submissions/" . $gradeable_id . "/" . $student_id . "/" . $display_version . "/" . $question_name;
```

Comment:
I overlooked this in my last review but you could also use `FileUtils::joinPaths()` for these paths instead of manually concatenating it.